.github/workflows/dependabot.yml

```
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: weekly
```

Maybe here too, it should be monthly

```suggestion
      interval: monthly
```